hbase-server/src/main/java/org/apache/hadoop/hbase/fs/HFileSystem.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.hadoop.hbase.fs;
 
+import static org.apache.hadoop.hbase.util.LocatedBlockHelper.*;
```

Comment:
```suggestion
import static org.apache.hadoop.hbase.util.LocatedBlockHelper.getLocatedBlockLocations;*;
```

Reply:
Seems the eclipse formatter spotless plugin will not expand the star imports automatically...